src/irmin/merge.ml

```diff
@@ -142,8 +142,7 @@ let option (type a) ((a, t) : a t) : a option t =
               | Some (Some o) ->
                   let pp = Type.pp a and ( = ) = Type.equal a in
                   Log.debug (fun f -> f "option old=%a" pp o);
-                  if x = o then ok (Some x) else conflict "option: add/del" ) )
-  )
+                  if x = o then ok (Some x) else conflict "option: add/del")) )
```

Comment:
the extra space looks strange here

Reply:
Multi-line tuples seem to get spaces even with `indicate-multiline-delimiters=no`.